source/draft_api.rst

```diff
@@ -209,21 +243,53 @@ Response body:
        "slugs":["qa", "test"]
     }
 
+Response body:
+
+.. code-block:: javascript
+
+    {
+       "name":"Quality Assurance/Testing",
+       "slugs":["qa", "test"],
+       "id": 1,
+       "deleted_at": null,
+       "parent": null
+    }
+
+
 *POST /times/*
 ~~~~~~~~~~~~~~
 
+Request body:
+
 .. code-block:: javascript
 
     {
       "duration":12,
       "user": "example-2",
-      "project": "",
+      "project": "Ganeti Web Manager",
       "activities": ["gwm", "ganeti"],
```

Comment:
These don't look like activities, they look like projects (and don't match activities in the response below)